apollo-router/src/query_planner/caching_query_planner.rs

```diff
@@ -107,6 +108,9 @@ where
         query_analysis: &QueryAnalysisLayer,
         cache_keys: Vec<WarmUpCachingQueryKey>,
     ) {
+        let _timer = Timer::new(|duration| {
+            ::tracing::info!(histogram.apollo.router.query_planning.duration = duration.as_secs_f64(), phase = %"warmup");
```

Comment:
Are we planning on adding query_planning.duration for non-warmup phase?

Reply:
we already have `apollo_router_query_planning_time` for that. The warm up phase calls into the normal process for planning, going through caches and bridge like a normal query. So if you have 3 queries to plan in warmup, you could get:
- apollo_router_query_planning_time = 3ms
- apollo_router_query_planning_time = 5ms
- apollo_router_query_planning_time = 4ms
- apollo.router.query_planning.duration = 12ms

We should not use a `phase` attribute to distinguish them, they are different metrics measuring different things